docs/src/snforge-advanced-features/fork-testing.md

```diff
@@ -21,37 +23,47 @@ We first need to define the contract's interface along with all the structures u
 
 There are two ways of configuring a fork:
 - by specifying `url` and block-related parameters in the `#[fork(...)]` attribute
-- or by passing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+- or by providing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+
+> 📝 **Note**
+> Using fork tests means `snforge` will make (often multiple) requests to the configured RPC URL.
+> These requests are relatively slow as they happen over the network.
```

Comment:
```suggestion
> Using fork tests means `snforge` will make (often multiple) requests to the configured RPC URL.
> This may lead to slowdown in tests due to their over-the-network nature.
```

Reply:
I prefer we kept the wording here simple. The fact is that fork tests are quite slow and users should expect that as normal.